jobs/kafka/spec

```diff
@@ -53,6 +53,10 @@ properties:
         replication_factor: 1
         partitions: 1
 
+  auto.create.topics.enable:
+    description: "Controls when Kafka enables auto creation of topic on the server."
+    default: true
```

Comment:
This changes the default behaviour of the bosh release. Could we start with just allowing the setting, and consider changing the default in future?

Reply:
I removed default section.
and rewrite `servers.properties`
please confirm it 👍

Reply:
And i wrote release notes.